src/http/websocket_http_client.zig

```diff
@@ -1636,12 +1622,12 @@ pub fn NewWebSocketClient(comptime ssl: bool) type {
             len: usize,
             op: u8,
         ) callconv(.C) void {
-            if (this.tcp.isClosed() or this.tcp.isShutdown()) {
+            if (this.tcp.isClosed() or this.tcp.isShutdown() or op > 0xF) {
```

Comment:
@cirospaciari is this right

Reply:
https://github.com/oven-sh/bun/blob/cff5dc0/src/http/websocket.zig
https://www.rfc-editor.org/rfc/rfc6455#section-5.2